tests/rust-integration-tests/integration_test/src/tests/seccomp_notify/seccomp_agent.rs

```diff
@@ -47,15 +54,13 @@ pub fn recv_seccomp_listener(seccomp_listener: &Path) -> SeccompAgentResult {
         Ok(msg) => msg,
         Err(e) => {
             let _ = unistd::close(conn);
-            let _ = unistd::close(socket);
             bail!("failed to receive message: {}", e);
         }
     };
 
     // We received the message correctly here, so we can now safely close the socket and connection.
     let _ = unistd::close(conn);
-    let _ = unistd::close(socket);
-
+    drop(socket);
```

Comment:
Can we add a message for why we are manually dropping the socket here in comment, so we don't accidentally remove it in future?

Reply:
I think L61 has already explained the reason, so is it necessary to add more comments?

Reply:
No, I think I got confused with something else. L61 Should be good enough :+1: